.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+defaults: &defaults
+  working_directory: /home/circleci/grafana-kentik-app
+  docker:
+    - image: circleci/node:10.9.0-jessie
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      #- run:
+      #    name: Update npm
+      #    command: 'sudo npm install -g npm@6.4.0'
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "package.json" }}
+      - run: yarn install
```

Comment:
it's better to use `--pure-lockfile` options, because we don't need update or install new packages here, just restore it from lockfile's state:
`yarn install --pure-lockfile --no-progress`